xarray/core/dask_array_ops.py

```diff
@@ -72,3 +80,66 @@ def push(array, n, axis):
             push, axis=axis, n=n, depth={axis: (1, 0)}, boundary="none"
         )
     return pushed
+
+
+def unify_chunks(*objects: "T_DSorDA") -> Tuple["T_DSorDA", ...]:
```

Comment:
I'm not sure this is the right place for this, usually the functions in `dask_array_ops` work at a lower level. `computation.py` (that's pretty big already, though) or `parallel.py`, maybe?

Reply:
Wouldn't work out of the box in `parallel.py` (circular import), so I went for `computation.py`